samples/web/content/apprtc/room.py

```diff
@@ -65,6 +65,27 @@ def has_room(host, room_id):
   room = memcache_client.get(key)
   return room is not None
 
+def remove_room(host, room_id, client_id):
+  memcache_client = memcache.Client()
```

Comment:
How can you use cas to do delete? If my understanding is correct, calling gets adds a timestamp like id to the object returned that is then used in the cas call to be sure the underlying record hasn't changed, but passing None as the value won't have that timestamp on it.

It seems like the only way we can use cas is to do a soft delete, with a deleted flag. 

Am I missing something?